index.html

```diff
@@ -49,8 +49,187 @@ <h1 class="username">I'm Mark, Glad to see you!</h1>
+                <div class="snapshoot-portfolio">
+                    <img src="./images/professional-art-printing.svg" alt="" class="portfolio-snapshoot">
+                </div>
+                <div class="left-block">
+                    <div class="project-summary">
+                        <h2 class="project-title">Multi-Post Stories</h2>
+                        <div class="summary-frame">
+                            <h3 class="client">CANOPY</h3>
+                            <h3 class="role">Back End Dev</h3>
+                            <h3 class="year">2015</h3>
+                        </div>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="tags">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="action">
+                        <button class="project-button" type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-myself">
+            <div class="skill-top">
+                <div class="name">
+                    <h2 class="username">About Myself</h2>
+                </div>
+                <div class="description">
+                    <p class="user-description">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                <h2 class="socialmedia">LET'S CONNECT</h2>
+                <div class="social-media-icons">
+                    <a href="https://twitter.com/Eng_Ngwala" class="twitter-link">
+                        <img src="./images/twitter.svg" alt="twitter" class="twitter" width="20" height="20">
+                    </a>
+                    <a href="https://www.linkedin.com/in/mark-otweyo-034752123/" class="linkedin-link">
+                        <img src="./images/linkedin.svg" alt="linkedin" class="linkedin" width="20" height="20">
+                    </a>
+                    <a href="mailto:markotweyo@gmail.com" class="email-link">
+                        <img src="./images/email.svg" alt="email" class="email" width="20" height="20">
+                    </a>
+                    <a href="https://github.com/Markotweyo" class="social-media-link">
+                        <img src="./images/github.svg" alt="github" class="github" width="20" height="20">
+                    </a>
+                </div>
+            </div>
```

Comment:
The semantics help the browser and SEO to interpret your code. Here, it is a list of links, that's why you should use `ul/li  `